### PR TITLE
Migrate to DI

### LINKED ISSUE #135: Migrate fusion-cli to DI

### PR DESCRIPTION
**Pull request by ganemone**
Fixes #135

### CODE REVIEW

**Comment by ganemone** [comment]
!merge

File: plugins/context-plugin.js [hunk 1]
```diff
@@ -4,34 +4,37 @@
 This is where new ctx properties should be initialized
 */
 
-const envVarsPlugin = require('./environment-variables-plugin');
-const compilationMetaDataPlugin = require('./compilation-metadata-plugin');
+const getEnv = require('./environment-variables-plugin');
```
Comment (AlexMSmithCA) [inline plugins/context-plugin.js]: Thoughts on removing `plugin` from the names here, to differentiate from the DI system?

Comment (ganemone) [reply plugins/context-plugin.js]: yea that sounds good

**Comment by ganemone** [comment]
!merge

**Pull request merged**